src/modules/configuration.js

```diff
@@ -0,0 +1,6 @@
+const gameKey = 'OOyBSMfV40z7yA2OR8pe';
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/OOyBSMfV40z7yA2OR8pe/scores';
+
+const apiEndPoint = `${baseURL}/games/${gameKey}/scores`;
```

Comment:
- [ ] It seems that you are passing your 'gameKey' and other URL information twice in your 'apiEndPoint' and this is preventing the user to connect with your application. Please fix this issue.
 -'Server Down':
<img width="1282" alt="image" src="https://user-images.githubusercontent.com/61919617/196807381-a08924ca-14dc-43a4-aa69-1e7e3993e1c6.png">

Reply:
@leonardodiasb thanks for the comments, I will fix it now